react-routing/client/src/App.js

```diff
@@ -0,0 +1,24 @@
+import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Catalog from './pages/Catalog';
+import ProductDetails from './pages/ProductDetails';
+import About from './pages/About';
+import NotFound from './pages/NotFound';
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
This `div` is OK and can be used to apply page styling, but it's not necessary. Usually the App would only contain routes.